src/Blocks/Query_Loop.php

```diff
@@ -0,0 +1,163 @@
+<?php declare(strict_types=1);
+
+namespace UCSC\Blocks\Blocks;
+
+use UCSC\Blocks\Blocks\Contracts\Taxonomies;
+use UCSC\Blocks\Blocks\Traits\With_Taxonomies;
+
+abstract class Query_Loop extends ACF_Group implements Taxonomies {
+    
+    use With_Taxonomies;
+	
+	public const QUERY_LOOP = 'query_loop';
+	public const QUERY_TYPE = 'query_type';
+	public const AUTOMATIC  = 'automatic';
+	public const MANUAL     = 'manual';
+	
+	public const AUTOMATIC_GROUP = 'automatic_group';
+	public const CATEGORIES      = 'categories';
+	
+	public const MANUAL_CARDS = 'manual_cards';
+	public const MANUAL_CARD  = 'manual_card';
+
+	/**
+	 * Default label for single card on Manual Query
+	 * Add/Change in Block clack in order to override
+	 */
+	protected string $default_manual_card_label = 'Card';
+
+	/**
+	 * Default max amount of cards applied to the manual query
+	 * Add/Change in Block clack in order to override
```

Comment:
What is "clack"?

Reply:
Is it supposed to be "class"? Also in the subsequent block comments.

Reply:
Updated